README.md

```diff
@@ -90,16 +77,10 @@ You can then make a request to the Lambda: `http://localhost:8888/api/v0.1/docs`
 
 ## Deployment
 
-Before deploying, ensure [configuration files](#configuration) have been properly set up:
-
-1. Copy `config/var_env.sample` to `config/dev.env`, `config/var_qa.env`, and `config/var_production.env`.
-   *  Verify environment variables are correct.
-2. Verify `.env` has correct settings for deployment.
-3. Verify `package.json` has correct command-line options for security group, VPC, and role (if applicable).
-4. Verify `config/event_sources_dev.json`, `config/event_sources_qa.json`, `config/event_sources_production.json` have proper event sources.
+Travis is set up to automatically deploy to the appropriate environment for development, qa, and production
```

Comment:
Can we say `", and production (master branch)"` - even though that's usually the case?

Reply:
Fixed